doc/src/fix_shake.rst

```diff
@@ -137,6 +137,13 @@ constrained (within a fudge factor of MASSDELTA specified in
 both bonds in the angle are constrained then the angle will also be
 constrained if its type is in the list.
 
+.. versionchanged:: TBD
+
+The types may be given as typelabels for as long as none of the
```

Comment:
rewording suggestion: "The types may be given as type labels only if there is no atom, bond or angle type label of *b*, *a*, *t*, or *m* defined in the simulation.  In this case, type label support is disabled because the type arguments could be ambiguous."

Reply:
I suggest this altered 2nd sentence in Jake's suggestion, if it is correct:

If that is the case, type labels cannot be used as constraint values with these 2 fixes because those type labels would mistakenly be treated as a new category of constraint.  Use numeric types instead.

Reply:
Done